corehq/apps/hqadmin/tests/test_authenticate_as.py

```diff
@@ -55,14 +55,6 @@ def test_invalid_data(self):
         form = AuthenticateAsForm(data)
         self.assertFalse(form.is_valid())
 
-    def test_no_login_as_other_webuser(self):
-        data = {
-            'username': 'awebuser'
-        }
-        form = AuthenticateAsForm(data)
-        self.assertFalse(form.is_valid())
-        self.assertTrue('not a CommCareUser' in str(form.errors))
```

Comment:
You could turn this into a test validating that logging in as a web user works

Reply:
reverted.